operators/src/pro/cache/cache_chunks.rs

```diff
@@ -186,11 +174,28 @@ where
         // So the query is not updated.
         Ok(())
     }
+
+    fn cache_element_hit(&self, query: &Self::Query) -> bool {
+        let temporal_hit = if let Some(time_bounds) = self.time_interval {
+            time_bounds == query.time_interval || time_bounds.intersects(&query.time_interval)
+        } else {
+            false // TODO: should we return true here?
```

Comment:
I think we have to think about if it makes sense that time_bounds can be None. But if they can, I think it has to be false here. As we do not know the bounds, we cannot guarantee that all elements of the query are contained in the element.

Reply:
it must be true because if it is the empty collection it might be the only empty in the cache and the cache must produce something.

Reply:
[75cf335](https://github.com/geo-engine/geoengine/pull/864/commits/75cf335a877d4817e8b2116cbb12990a519fff5a)

Reply:
[d8ac3d3](https://github.com/geo-engine/geoengine/pull/864/commits/d8ac3d359f68ad85ada90e3ef513af689250dbbe)